luigi/contrib/bigquery_avro.py

```diff
@@ -74,6 +75,13 @@ def read_schema(fp):
             raise exception_reading_schema[0]
         return schema[0]
 
+    @staticmethod
+    def get_writer_schema(datum_reader):
+        if sys.version_info >= (3, 0):
```

Comment:
I think that that `if six.PY3L:` is much more clear

Reply:
Sure.